pkg/backends/tcp.go

```diff
@@ -82,9 +91,10 @@ func (b *TCPListener) Addr() net.Addr {
 }
 
 // Start runs the given session function over the WebsocketListener backend
-func (b *TCPListener) Start(bsf netceptor.BackendSessFunc, errf netceptor.ErrorFunc) {
+func (b *TCPListener) Start(shutdownContext context.Context, wg sync.WaitGroup, bsf netceptor.BackendSessFunc, errf netceptor.ErrorFunc) {
 	var err error
-	b.li, err = net.Listen("tcp", b.address)
+	var listenerCfg net.ListenConfig
+	b.li, err = listenerCfg.Listen(shutdownContext, "tcp", b.address)
```

Comment:
I'm not sure we actually need this, the docs on what this context actually does are lacking https://golang.org/pkg/net/#ListenConfig